src/components/ObsDetails/DetailsTab.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+
+import {
+  Body2,
+  Body4,
+  Button,
+  DateDisplay,
+  Divider,
+  Heading4,
+  ObservationLocation,
+  QualityGradeStatus
+} from "components/SharedComponents";
+import KebabMenu from "components/SharedComponents/KebabMenu";
+import Map from "components/SharedComponents/Map";
+import { View } from "components/styledComponents";
+import { t } from "i18next";
+import _ from "lodash";
+import type { Node } from "react";
+import React, { useState } from "react";
+import { Menu } from "react-native-paper";
+import IconMaterial from "react-native-vector-icons/MaterialIcons";
+import useIsConnected from "sharedHooks/useIsConnected";
+
+import Attribution from "./Attribution";
+import checkCamelAndSnakeCase from "./helpers/checkCamelAndSnakeCase";
+
+type Props = {
+  observation: Object
+}
+
+const headingClass = "mt-[20px] mb-[11px] text-black";
+const sectionClass = "mx-[15px] mb-[20px]";
+
+const DetailsTab = ( { observation }: Props ): Node => {
+  const isOnline = useIsConnected( );
+  const application = observation?.application?.name;
+  const [locationKebabMenuVisible, setLocationKebabMenuVisible] = useState( false );
+
+  const displayQualityGradeOption = option => {
+    const qualityGrade = observation?.quality_grade;
+    const labelClassName = ( qualityGrade === option ) ? "font-bold" : "";
+
+    return (
+      <View className="flex-col space-y-[8px]">
+        <QualityGradeStatus qualityGrade={option} />
+        <Body4 className={labelClassName}>{_.startCase( _.camelCase( t( option ) ) )}</Body4>
```

Comment:
This needs to be under translator control. In general, if you find yourself using `startCase()` or `toUpperCase()` on text the user sees, you probably should be globalizing it instead, so here you probably want something like

```javascript
switch ( option ) {
  case "research":
    return t( "quality_grade_Research" );
  case "needs_id":
    return t( "quality_grade_Needs_ID" );
  default:
    return t( "quality_grade_Casual" );
}
```

Note that the keys provide context (we're talking about "research" the quality grade, not "research" the second person imperative verb), and that they specify the desired case, which translators can override if they want.